app/ui/src/app/integration/import-export/import/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './integration-import.component';
+export * from './integration-import.module';
+export * from './integration-import.actions';
+export * from './integration-import.models';
+export * from './integration-import.reducer';
```

Comment:
You're already importing and bootstrapping the Reducer at `integration-import.module`, hence there's no need to export it from here. all the way around, this might end up in a circular reference. Same applies to `integration-import.component` above.